gitman/git.py

```diff
@@ -117,7 +117,7 @@ def valid():
     currentdir = pwd(_show=False)
 
     status = False
-    if gittoplevel[0] == currentdir:
+    if os.path.normpath(os.path.normcase(gittoplevel[0])) == os.path.normpath(os.path.normcase(currentdir)):
         status = True
     else:
         log.debug(
```

Comment:
I think we should print the path values _after_ conversion to 'normpath' and 'normcase' so that the user can see what the program is actually checking against. In that case the normpath/normcases should be moved to where the variables _gittoplevel_ and _currentdir_ are defined.